ui/src/app/app-routing.tsx

```diff
@@ -1,20 +1,32 @@
 import {Component as AComponent} from '@angular/core';
 import {AppRoute, AppRouter} from 'app/components/app-router';
+import {CookiePolicy} from 'app/pages/cookie-policy';
+import {SessionExpired} from 'app/pages/session-expired';
+import {SignInAgain} from 'app/pages/sign-in-again';
+import {UserDisabled} from 'app/pages/user-disabled';
+import {SignInService} from 'app/services/sign-in.service';
 import { ReactWrapperBase } from 'app/utils';
 import * as React from 'react';
-import {CookiePolicyComponent} from './pages/cookie-policy';
 
-export const AppRoutingComponent: React.FunctionComponent = () => {
+export const AppRoutingComponent: React.FunctionComponent<{signIn: Function}> = ({signIn}) => {
   return <AppRouter>
-    <AppRoute path='/cookie-policy' component={CookiePolicyComponent}/>
+    <AppRoute path='/cookie-policy' component={CookiePolicy}/>
+    <AppRoute path='/session-expired' data={{signIn: signIn}} component={SessionExpired}/>
+    <AppRoute path='/sign-in-again' data={{signIn: signIn}} component={SignInAgain}/>
+    <AppRoute path='/user-disabled' component={UserDisabled}/>
   </AppRouter>;
 };
 
 @AComponent({
   template: '<div #root></div>'
 })
 export class AppRouting extends ReactWrapperBase {
-  constructor() {
-    super(AppRoutingComponent, []);
+  constructor(private signInService: SignInService) {
```

Comment:
This was originally consumed in the Angular wrappers for `SignInAgain` and `SessionExpired`. Pushing it down here so that 1) it's in one place instead of several and 2) so that I don't have to figure out how to deprecate it before we get rid of the Angular router.